content/docs/tutorials/login_requiring/login@es.md

```diff
@@ -0,0 +1,78 @@
+---
+$title: Iniciar Sesión
+$order: 0
+---
+
+La primera vez que aterrizas en la página, puedes ver 2 comentarios y un botón de inicio de sesión.
+
+<amp-img src="/static/img/login-button.png" alt="Botón de inicio de sesión" height="290" width="300"></amp-img>
+
+Si busca el botón de inicio de sesión en el código, encontrará:
+
+[sourcecode:html]
+<span amp-access="NOT loggedIn" role="button" tabindex="0" amp-access-hide>
```

Comment:
This code should remain in English so it's consistent with other code samples and with the images.  Consider changing it back to English.

Reply:
Done

Reply:
Thanks!